service/lib/dinstaller/luks_activation_question.rb

```diff
@@ -29,22 +29,28 @@ module DInstaller
   #   * decrypt: to activate the device using the provided password
   #
   # @example
-  #   question = LuksQuestion.new("/dev/sda1", label: "mydata", size: "10 GiB")
+  #   question = LuksActivationQuestion.new("/dev/sda1", label: "mydata", size: "10 GiB")
   #   question.password = "n0ts3cr3t"
   #
   #   question.answer = :skip    # in case you do not want to activate the device
   #
   #   question.answer = :decrypt # in case you want to decrypt with the given password
-  class LuksQuestion < Question
+  class LuksActivationQuestion < Question
+    # Current attempt to decrypt the device
+    #
+    # @return [Integer]
+    attr_reader :attempt
```

Comment:
Because the client needs it to know whether the same question was asked again. For example, when the provided encryption password does not work, a new identical questions is created, but with attempt+1.